integrations/weaviate/tests/test_document_store.py

```diff
@@ -35,6 +35,7 @@
 )
 
 
+@pytest.mark.integration
```

Comment:
not all the following tests are integration tests, but I chose not to interfere much with Weaviate, since it is a work in progress.